src/Symfony/Component/Config/Tests/Definition/Builder/NodeDefinitionTest.php

```diff
@@ -60,10 +68,50 @@ public function testDocUrlWithoutPackage()
 
     public function testUnknownPackageThrowsException()
     {
+        $node = new ArrayNodeDefinition('node');
```

Comment:
Moving `expectException` just before the call that produces it to be more specific.